tools/travis/production-helper

```diff
@@ -87,7 +87,7 @@ fi
 echo; echo "Now confirming all the RabbitMQ queue processors are correctly registered!"; echo
 # These hacky shell scripts just extract the sorted list of queue processors, running and expected
 supervisorctl status | cut -f1 -dR | cut -f2- -d: | grep events | cut -f1 -d" " | cut -f3- -d_ | cut -f1 -d- | sort -u > /tmp/running_queue_processors.txt
-su zulip -c /home/zulip/deployments/current/scripts/lib/queue_workers.py | grep -v ^test$ > /tmp/expected_queue_processors.txt
+su zulip -c /home/zulip/deployments/current/scripts/lib/queue_workers.py | grep -v ^test$ | sort -u > /tmp/expected_queue_processors.txt
```

Comment:
Shell command 'sort' depends on system locale and symbol '_' (underscore) is located after letters in the default locale of travis instances. Otherwise python sorting uses its own simbols ordering and underscore is located before letters. I've tried to change locale for sort shell command through changing environment variable which works on my side but it doesn' t work on travis instances. That's why I've decided to apply the same sorting command to both comparing lists.